src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -810,4 +810,19 @@ This package has not been loaded because another conflicting package was loaded
   <data name="NodeUnhandledMsg" xml:space="preserve">
     <value>Unhandled 'DummyNode.NodeNature' value: {0}</value>
   </data>
+  <data name="HideWiresPopupMenuItem" xml:space="preserve">
+    <value>Hide Wires</value>
+  </data>
+  <data name="UnhideWiresPopupMenuItem" xml:space="preserve">
+    <value>Unhide Wires</value>
+  </data>
```

Comment:
I think I intentionally removed these two

Reply:
Removed for now, will get these once #12050 is merged :)